schema/proto3/port.proto

```diff
@@ -28,13 +28,14 @@ message PortConfiguration {
     
     NetworkType network_type = 2;
     string project_id = 3;
-    string network_id = 4;
-    string id = 5;
-    string name = 6;
-    string network_ns = 7;
-    string mac_address = 8;
-    string veth_name = 9;
-    bool admin_state_up = 10;
+    string subnet_id = 4;
+    uint32 mq_subscription_index = 5;
```

Comment:
This new field would work for Kafka. How about RocketMQ or Pulsar? @haboy52581 @er1cthe0ne

Reply:
this is the last blocking issue, I think we can merge this PR once we have answer on this.

Reply:
mq_subscription_index has been removed per discussion.